speechmatics/models.py

```diff
@@ -194,6 +194,11 @@ class SummarizationConfig:
     """Optional summarization summary_type parameter."""
 
 
+@dataclass
+class BatchSentimentAnalysisConfig:
```

Comment:
SentimentAnalysisConfig should be sufficient. Later on we can have Batch as subclass, if necessary.

Reply:
done